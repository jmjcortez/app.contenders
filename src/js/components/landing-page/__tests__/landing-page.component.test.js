import React from 'react';
import LandingPage from '../landing-page.component';
import { shallow } from 'enzyme';

describe('Landing page test', () => {
    it('should render correctly with no props', () => {
        const component = shallow(<LandingPage />);
        
        expect(component).toMatchSnapshot();
    });

    it('should redirect when user is authenticated', () => {
        
    });
});