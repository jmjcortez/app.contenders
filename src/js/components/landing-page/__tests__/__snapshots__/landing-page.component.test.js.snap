// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Field input test should render correctly with no props 1`] = `
<div>
  <Styled(Navbar)>
    <Styled(NavbarBrand)
      className="mx-auto"
    >
      CONTENDERS
    </Styled(NavbarBrand)>
    <Nav
      fill={false}
      justify={false}
    >
      <Styled(NavbarBrand)
        href="login"
        login={true}
      >
        Login
      </Styled(NavbarBrand)>
      <Styled(NavbarBrand)
        href="register"
      >
        Register
      </Styled(NavbarBrand)>
    </Nav>
  </Styled(Navbar)>
  <Hero />
</div>
`;

exports[`Landing page test should render correctly with no props 1`] = `
<div>
  <Styled(Navbar)>
    <Styled(NavbarBrand)
      className="mx-auto"
    >
      CONTENDERS
    </Styled(NavbarBrand)>
    <Nav
      fill={false}
      justify={false}
    >
      <Styled(NavbarBrand)
        href="login"
        login={true}
      >
        Login
      </Styled(NavbarBrand)>
      <Styled(NavbarBrand)
        href="register"
      >
        Register
      </Styled(NavbarBrand)>
    </Nav>
  </Styled(Navbar)>
  <Hero />
</div>
`;
